robot moved from countertop to cabinet
robot picks up knife from cabinet
robot moved from cabinet to countertop
robot puts down knife to countertop
robot moved from countertop to cabinet
robot picks up bowl from cabinet
robot moved from cabinet to countertop
robot puts down bowl to countertop
carrot already sliced!
robot picks up butter from countertop
robot moved from countertop to stove
robot puts down butter to pan
waiting for 1 minutes
robot moved from stove to countertop
robot picks up sliced_carrot from countertop
robot moved from countertop to stove
robot puts down carrot to pan
contents of pan are ['butter', 'carrot']
contents of pan are mixed!
robot moved from stove to countertop
robot picks up milk from countertop
robot moved from countertop to stove
robot puts down milk to pan
Traceback (most recent call last):
  File "/home/shivam/Desktop/side_quests/generalisation_project/OKG/frameworks/GPT4/streamlined_for_experiments/for_knowledge_expansion_2/./LLM_only/final_run.py", line 16, in <module>
    put_down_obj("sugar", "pan")
  File "/home/shivam/Desktop/side_quests/generalisation_project/OKG/frameworks/GPT4/streamlined_for_experiments/for_knowledge_expansion_2/LLM_only/functions/actions.py", line 442, in put_down_obj
    raise RuntimeError(f"put_down_obj({obj}, {loc}) #{obj} not present in the environment!")
RuntimeError: put_down_obj(sugar, pan) #sugar not present in the environment!
